pandas/core/ops/__init__.py

```diff
@@ -474,11 +475,57 @@ def dispatch_to_series(left, right, func, str_rep=None, axis=None):
     """
     # Note: we use iloc to access columns for compat with cases
     #       with non-unique columns.
+    eval_kwargs = eval_kwargs or {}
+
     import pandas.core.computation.expressions as expressions
 
     right = lib.item_from_zerodim(right)
+
     if lib.is_scalar(right) or np.ndim(right) == 0:
 
+        new_blocks = []
+        mgr = left._data
+        for blk in mgr.blocks:
+            # Reshape for EA Block
+            blk_vals = blk.values
```

Comment:
Should this be using `to_numpy` instead of `.values`? Not super well versed on what types actually get preserved this way

Reply:
we specifically want to get the .values attribute, which can be either a ndarray or EA (also I dont think Block has to_numpy)